mrbelvedereci/build/handlers.py

```diff
@@ -16,8 +17,9 @@ def queue_build(sender, **kwargs):
         return
 
     # Queue the pending status task
-    res_status = set_github_status.delay(build.id)
-    build.task_id_status_start = res_status.id
+    if settings.GITHUB_STATUS_UPDATES_ENABLED:
+        res_status = set_github_status.delay(build.id) 
+        build.task_id_status_start = res_status.id
```

Comment:
nope, they are just for managing the child task, which won't be created in this case.